### PR TITLE
Return 400 if attempts too high

### PR DESCRIPTION
**Pull request by StevenLeighton21**
Add different return status if attempts too high, change attempts to integer with default 0

This will let the front end show the right error if there have been too many attempts

### CODE REVIEW

**Review by njseeto — COMMENTED** [review]
Looks good - one question about the db migration.

File: db/migrate/20230329132744_create_saved_forms.rb [hunk 1]
```diff
@@ -10,7 +10,7 @@ def change
       t.string :user_id
       t.string :user_token
       t.text :user_data_payload
-      t.numeric :attempts, default: 0
+      t.integer :attempts, default: 0
```
[the remark this answers is not in the document]
Comment (StevenLeighton21) [reply db/migrate/20230329132744_create_saved_forms.rb:13]: There's no saved form table in the schema yet so it should re-apply, it does locally but you might be right about the behaviour in dev, will split out into a change migration

File: db/migrate/20230424105345_change_saved_forms_attempts_to_integer.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class ChangeSavedFormsAttemptsToInteger < ActiveRecord::Migration[6.1]
+  def change
+    change_column :saved_forms, :attempts, :integer
```
Comment (njseeto) [inline db/migrate/20230424105345_change_saved_forms_attempts_to_integer.rb:3]:
```suggestion
    change_column :saved_forms, :attempts, :integer, default: 0
```

**Review by njseeto — COMMENTED** [review]
One small comment, otherwise looks good!

**Pull request merged**